app/api/events.py

```diff
@@ -64,23 +111,30 @@ class Meta:
                            many=True,
                            type_='sponsor')
 
+    user = Relationship(attribute='user',
+                        self_view='v1.event_user',
+                        self_view_kwargs={'id': '<id>'},
+                        related_view='v1.user_detail',
+                        related_view_kwargs={'id': '<id>'},
+                        schema='UserSchema',
+                        type_='user')
+
 
 class EventList(ResourceList):
-    decorators = (jwt_required, )
-    schema = EventSchema
-    data_layer = {'session': db.session,
-                  'model': Event}
 
+    def before_create_object(self, data, view_kwargs):
+        data['starts_at'] = data['starts_at'].replace(tzinfo=None)
+        data['ends_at'] = data['ends_at'].replace(tzinfo=None)
```

Comment:
@magdalenesuo 
I still think it is saving it correctly . 

See this ( It is a screenshot of database from Valentino Studio ) :

![image](https://cloud.githubusercontent.com/assets/17861054/26769066/8dc46cd6-49c9-11e7-882c-6aff01a849c1.png)

Can you confirm as to where are you getting timezone aware time ?

I also did get a timezone aware response when I was using Postman ( google chrome app , for testing ) but that is not being stored.